onnxruntime/python/tools/transformers/models/bert/eval_squad.py

```diff
@@ -254,11 +261,22 @@ def parse_arguments(argv=None):
         help="Optional onnx model path. If not specified, optimum will be used to export onnx model for testing.",
     )
 
-    parser.add_argument("--use_gpu", required=False, action="store_true", help="Use CUDA execution provider.")
-    parser.set_defaults(use_gpu=False)
+    parser.add_argument("--use_gpu", required=False, action="store_true", help="Allow GPU RUN.")
```

Comment:
How about removing use_gpu since we can deduce it from value of --provider?

Reply:
Sure I can update that.

Reply:
@TedThemistokleous Could please remove this flag? Also all the `use_gpu` logic and opt for `provider`.